packages/api/src/schema.ts

```diff
@@ -2665,7 +2665,7 @@ const schema = gql`
   union MoveToFolderResult = MoveToFolderSuccess | MoveToFolderError
 
   type MoveToFolderSuccess {
-    articleSavingRequest: ArticleSavingRequest!
```

Comment:
@jacksonh I have replaced the `articleSavingRequest` with `success` here as a response